core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -236,6 +236,10 @@ private[spark] class LiveListenerBus(conf: SparkConf) {
     queues.asScala.map(_.name).toSet
   }
 
+  // For testing only.
+  private[scheduler] def getQueueCapacity(name: String): Int = {
```

Comment:
nit: could you change the return type to `Option[Int]` and use `None` to indicate an unknown queue rather than `-1`?

Reply:
changed